lib/src/Pos/Launcher/Resource.hs

```diff
@@ -171,8 +149,6 @@ allocateNodeResources transport networkConfig np@NodeParams {..} sscnp txpSettin
         dlgVar <- mkDelegationVar
         logDebug "Created DLG var"
         sscState <- mkSscState
-        logDebug "Created SSC var"
```

Comment:
This line seems to be removed by mistake.

Reply:
Right, will put it back.